tests/tap_tester/test_mixpanel_automatic_fields.py

```diff
@@ -65,8 +65,3 @@ def test_standard_auto_fields(self):
         """Automatic fields test for standard server"""
         self.eu_residency = False
         self.automatic_fields_test_run()
```

Comment:
Instead of removing SAA test, we should remove api_secret tests to reduce the execution time since api_secret it already deprecated.